contrib/cirrus/lib.sh

```diff
@@ -257,3 +260,79 @@ execute_local_registry() {
 
     verify_local_registry
 }
+
+setup_rootless() {
+    req_env_vars GOPATH GOSRC SECRET_ENV_RE
+
+    ROOTLESS_USER="rootlessuser"
```

Comment:
This line is unnecessary and might mislead future maintainers.  The top-level definition is perfectly fine.